src/panels/lovelace/cards/hui-sensor-graph-card.js

```diff
@@ -0,0 +1,273 @@
+import { LitElement, html } from '@polymer/lit-element';
+
+import '../../../components/ha-card.js';
+import '../../../components/ha-icon.js';
+
+import computeStateName from '../../../common/entity/compute_state_name.js';
+import stateIcon from '../../../common/entity/state_icon.js';
+
+import EventsMixin from '../../../mixins/events-mixin.js';
+
+class HuiSensorGraphCard extends EventsMixin(LitElement) {
+  set hass(hass) {
+    this._hass = hass;
+    const entity = hass.states[this.config.entity];
+    if (entity && this.entity !== entity) {
+      this.entity = entity;
+      this._getHistory();
+    }
+  }
+
+  static get properties() {
+    return {
+      _hass: Object,
```

Comment:
We should not use Object as type in Lit. Instead use `{ }`. It will default to the String constructor, which is just as bad. This is something that we will eventually also enforce with [this lint rule](https://github.com/43081j/eslint-plugin-lit/blob/0f01639b826dc8486b7e52ceeff663a807178a6a/docs/no-object-property-type.md) (but the lint rule is not compatible with Polymer 3 yet)